app/web_framework/api_version_handler.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+
+class APIVersionHandler():
+    OLD_VERSIONED_URI_COMPONENT = 'v1'
+    API_VERSION_HEADER_KEY = 'Api-Version'
+
+    _versions = [
+        1,
+        2
+    ]
+
+    @classmethod
+    def resolve_version(cls, accept_header_value: str, uri: str) -> int:
```

Comment:
Looks like `accept_header_value` is optional. (`accept_header_value: Optional[str]`)